hack/test-integration.sh

```diff
@@ -65,18 +125,23 @@ function cleanup {
 
 }
 
-if [[ "$SKIP" != "--skip-test" ]]; then
+parseArgs "$@"
+
+if [[ -z $SKIP_CLEANUP ]]; then
   trap cleanup EXIT
 fi
 
-if [[ "$SKIP" != "--skip-build" ]]; then
+if [[ -z $SKIP_BUILD ]]; then
   buildImage
+  verifyForBuildSuccess
 fi
 
 deployOperator
 
 waitForDeployment
+verifyPodIsSuccessfull
 
-if [[ "$SKIP" != "--skip-test" ]]; then
+if [[ -z $SKIP_TEST ]]; then
   runTests
+  verifyPodIsSuccessfull
 fi
```

Comment:
nit, I would maybe just use one verifyPodIsSuccessfull
in the end, that gets always called. Otherwise, we have duplicated log lines.
The naming is maybe also a bit confusing, since it is not verifying anything but printing the pod status